### ISSUE TITLE
Explore loss functions

### ISSUE THREAD

**Issue by mathiashusted** [gromdimon/brain-segment#24]
Check docs/optimizations.md

**Comment by gromdimon** [comment]
Our docs: https://github.com/gromdimon/brain-segment/blob/main/docs/optimizations.md#loss-function

We need to:
run classical training with each of the loss functions: https://docs.monai.io/en/stable/losses.html#
compare results on W&B

**Comment by mathiashusted** [comment]
Dice most likely OK
